src/components/BottomControls/BottomControls.jsx

```diff
@@ -1,15 +1,53 @@
 import React from "react";
-import FilterStatus from "../FilterStatus/FilterStatus";
+import { useSelector, useDispatch } from "react-redux";
+import { StatusFilters } from "../../redux/reducerFilter";
 import styles from "./style.module.scss";
 
-function BottomControls() {
+const RemainingTodos = ({ count }) => {
```

Comment:
use only one component in file. If you want to divide it^create a separated file for this

Reply:
OK,  I have implemented this